src/hooks.h

```diff
@@ -0,0 +1,78 @@
+/*
+* Copyright (C) the libgit2 contributors. All rights reserved.
+*
+* This file is part of libgit2, distributed under the GNU GPL v2 with
+* a Linking Exception. For full terms see the included COPYING file.
+*/
+
+#ifndef INCLUDE_hooks_h__
+#define INCLUDE_hooks_h__
+
+#include "git2/hooks.h"
+
+#define GIT_HOOK_FALSE 0
+#define GIT_HOOK_TRUE 1
```

Comment:
Let's not redefine booleans. We already have `true` and `false` available internally. For external code, we use an `int` directly with `0` or `1`.